### DISCUSSION TITLE
[BUG] Queue Messages - Never(Other messages cannot be displayed on the panel and need to wait for the set resend of 5 times to end before receiving other messages)

### DISCUSSION THREAD

**Discussion by anei20018cool** [nextgenhealthcare/connect#6363] [Q&A]
Version  4.1.1

I have an HTTP Sender destination   .Set it to Queue Messages - Never and then set the Retry Count to 5,Other messages cannot be displayed on the panel and need to wait for the set resend of 5 times to end before receiving other messages

**Comment by pacmano1** [comment]
This is your misunderstanding of how Mirth works.

**Comment by jonbartels** [comment]
https://docs.nextgen.com/bundle/Mirth_User_Guide_4_5_0/page/connect/connect/topics/c_Destination_Settings_connect_ug.html

and

https://docs.nextgen.com/bundle/Mirth_User_Guide_4_5_0/page/connect/connect/topics/c_Advanced_Queue_Settingsconnect_ug.html 

and

https://docs.nextgen.com/search?bundle=Mirth_User_Guide_4_5_0&q=thread

Are all important for the issue you have. With no queueing, messages are sent one at a time in a single thread _regardless_ of retry strategies.
